ftl/core/browsing.ftl

```diff
@@ -1,6 +1,8 @@
 browsing-add-notes = Add Notes...
 browsing-add-tags = Add Tags
 browsing-add-tags2 = Add Tags...
+browsing-add-to-selected-notes = Add to selected Notes
```

Comment:
```suggestion
browsing-add-to-selected-notes = Add to Selected Notes
```

Reply:
I think a capital there is probably more natural. We could also potentially use "Add to Selected" to make it a bit less verbose, but no strong feelings there either way.

Reply:
I actually wrote "Add to Selected" at first, because that's already a comparatively long label. But since, at the moment, it's the only sidebar action that interacts with the table selection (instead of the sidebar selection), I thought it might be necessary to be a little more detailed.